libs/go/libp2p_node/dht/dhtclient/dhtclient.go

```diff
@@ -688,8 +707,8 @@ func (dhtClient *DHTClient) RouteEnvelope(envel *aea.Envelope) error {
 		return err
 	}
 
-	if status.Code != dhtnode.Status_SUCCESS {
-		err = errors.New(status.Code.String() + " : " + strings.Join(status.Msgs, ":"))
+	if statusPerformative.Body.Code != SUCCESS {
```

Comment:
The extra `Body` is now needed as currently the generator is restricted to either fully generate a message or take it fully from custom spec.